### PR TITLE
adding property info

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Adding request_object_signing_alg property info
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-254606](https://oktainc.atlassian.net/browse/OKTA-254606)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -5011,13 +5012,15 @@ Applications have the following properties:
 | credentials        | credentials for the specified `signOnMode`     | [Application Credentials Object](#application-credentials-object)    | TRUE         | FALSE      | FALSE        |               |             |
 | settings           | settings for app                               | Object ([App Names & Settings](#app-names--settings))                | TRUE         | FALSE      | FALSE        |               |             |
 | profile            | Valid JSON schema for specifying properties    | [JSON](#profile-object)                                              | TRUE         | FALSE      | FALSE        |               |             |
+| request_object_signing_alg| The type of JSON Web Key Set (JWKS) algorithm that must be used for signing request objects. | `HS256`, `HS384`, `HS512`, `RS256`, `RS384`, `RS512`, `ES256`, `ES384`, `ES512`  | TRUE      | FALSE     | FALSE      |
```
Comment (danielzeller-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: so it's fine to omit the last 2 `|`?

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -5011,13 +5012,15 @@ Applications have the following properties:
...
 | _links             | discoverable resources related to the app      | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)       | TRUE         | FALSE      | TRUE         |               |             |
 | _embedded          | embedded resources related to the app          | [JSON HAL](http://tools.ietf.org/html/draft-kelly-json-hal-06)       | TRUE         | FALSE      | TRUE         |               |             |
 
 Property details
 
  * `id`, `created`, `lastUpdated`, `status`, `_links`, and `_embedded` are only available after an app is created.
  * `profile` is only available for OAuth 2.0 client apps. See [Profile Object](#profile-object).
+ * When you specify a value for the `request_object_signing_alg` property, all request objects from the client must be rejected if not signed with the specified algorithm. The algorithm must be used both when the request object is passed by value (using the request parameter) and when it is passed by reference (using the `request_uri` parameter). If a value for `request_object_signing_alg` isn't specified, the default is any algorithm that is supported by both the client and the server.
```
Comment (danielzeller-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: nit: "all request objects from the client *must be* rejected if not signed" -> "*will be* rejected if not signed" 

also, we do not support `request_uri` today, so probably better to omit it from the docs.

File: packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md [hunk 3]
```diff
@@ -811,13 +812,15 @@ Property Details
...
-    value that includes `authorization_code` implies a `response_types` value that includes `code`, as both values are defined as part of
-    the OAuth 2.0 authorization code grant.
+  endpoints in the [OAuth 2.0 protocol](https://tools.ietf.org/html/rfc6749). However, they are related in that the `grant_types`
+  available to a client influence the `response_types` that the client is allowed to use, and vice versa. For instance, a `grant_types`
+  value that includes `authorization_code` implies a `response_types` value that includes `code`, as both values are defined as part of
+  the OAuth 2.0 authorization code grant.
+
+* When you specify a value for the `request_object_signing_alg` property, all request objects from the client must be rejected if not signed with the specified algorithm. The algorithm must be used both when the request object is passed by value (using the request parameter) and when it is passed by reference (using the `request_uri` parameter). If a value for `request_object_signing_alg` isn't specified, the default is any algorithm that is supported by both the client and the server.
```
Comment (danielzeller-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md]: same comment as above

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: they're there..it renders properly when I build locally.

**Pull request merged**